pywbem_mock/_wbemconnection_mock.py

```diff
@@ -119,6 +110,244 @@ def _cvt_obj_name(objname):
+        implementation provided with this call as the `provider` parameter is
+        to be executed as the request response method for the namespaces
+        defined in the `namespaces` parameter, the provider type defined in the
+        provider 'provider_type` attribute of the `provider` and the classes
+        defined in the provider `provider_classnames` attribute of the
+        `provider`.
+
+        The provider registration process includes:
+
+        1. Validation that the namespaces defined for the provider exist.
+        2. Validation that the superclass of the provider is consistent with
+           the `provider_type` attribute defined in the provider.
+        3. Installation of any CIM classes defined by the provider
+           (`provider_classnames` attribute) including installation of
+           dependencies for these classes using the `schema_pragma_files` to
+           locate the search directories for dependencies.
+        4. Adding the provider to the registry of user_providers so that any
+           of the request methods defined for the `provider_type` are
+           passed to this provider in place of the default request processors.
+
+        Providers can only be registered for the following request response
+        methods:
+
+        1. provider_type = 'instance': CreateInstance, ModifyInstance, and
+           DeleteInstance
+
+        2. provider_type = 'method': InvokeMethod.
+
+        Each classname in a particular namespace may have at most one provider
+        registered
+
+        Parameters:
+
+          conn  (:class:`~pywbem_mock.FakedWBEMConnection`):
+            Defines the attributes of the connection. Used to issue requests to
+            create instances in the interop namespace for all existing
+            namespaces that do not have instances of CIM_Namespace defined
+
+          provider (instance of subclass of :class:`pywbem_mock:InstanceWriteProvider` or :class:`pywbem_mock:MethodProvider`):
+            The methods in this subclass override the corresponding methods in
+            the superclass. The method call parameters must be the same
+            as the default method in the superclass and it must return
+            data in the same format if the default method returns data.
+             This
+            class must contain variables `provider_type` and
+            `provider_classnames` that define the type of provider and the CIM
+            classes that the provider serves.
+
+          namespaces (:term:`string` or :class:`py:list` of :term:`string`):
+            Namespace or namespaces for which the provider is being registered.
+
+            If `None`, the default namespace of the connection will be set to
+            the built-in default namespace
+.
+          schema_pragma_files (:term:`py:iterable` of :term:`string` or :term:`string`):
+            File paths defining a schema pragma file MOF for the set of CIM
```

Comment:
Suggest to change this to: "Path names of schema pragma files for the set of CIM ..."

Reply:
DONE